### DISCUSSION TITLE
[Help wanted] - Remote moderated usability test on the way we explain and show exchange data

### DISCUSSION THREAD

**Discussion by Alportan** [electricitymaps/electricitymaps-contrib#6687] [General]
Hi, 👋

We're excited to invite you to a casual usability test to help us improve our app. Your input has always been super helpful, and in this session, we'll be focusing on the way we explain and display exchange data, along side a few fresh design updates.

**Details:**
Duration: Around 45 minutes
Format: Remote moderated usability test (on a prototype), via Google Meet 

We'd love to hear your thoughts on this! If you're up for it, just schedule a time that fits you best via [Google Calendar here](https://calendar.app.google/XSUSDBMVVVQrHdCQ6). 👈 

Thanks a bunch for your ongoing support. Looking forward to catching up and brainstorming together! 🚀

**Comment by andiz2** [comment]
Hi!
Only for me the link of Google Calendar looks broken?

500. That’s an error.

There was an error. Please try again later. That’s all we know.

**Comment by Alportan** [comment]
@andiz2 thanks for your interest!

That's odd, I just tested the link again and it works on my side! 

As an alternative, you can send me an email to alex.portan@gmail.com and we can book an appointment from there. 🙌
